pro-planner-client/src/routes/LandingPage.jsx

```diff
@@ -30,8 +30,7 @@ const LandingPage = () => {
 
         const [showTripTooltip, setShowTripTooltip] = useState(false);
         const [showOutingTooltip, setShowOutingTooltip] = useState(false);
-        // may remove the localhost regex once deployed
-        const planUrlRegex = /^((https?:\/\/[^/]+\/[^/]+)|(https?:\/\/localhost:3000\/[^/]+))$/;
+        const planUrlRegex = /^https?:\/\/(?:[^\/:]+[:.])?[^\/:]+\/[^/]+$/;
```

Comment:
I don't think removal is the best option. I think it's possible to construct a regex expression using the RegExp constructor ([reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)) to intergrate strings from variables into it.

Reply:
Oh, the reason I added the `localhost:3000` regex expression is to bypass the Invalid Plan URL format error during the local test since ":" in the local domain threw the Invalid Plan URL error although the URL is correct(ex. http://localhost:3000/abc). So I just added that extra expression in my first commit. But I updated the regex expression so that it won't throw error for the correct local domain url without the extra `localhost:3000` expression. Also, once the app is deployed, we wouldn't need the local domain right?

Reply:
Ah I see. Yeah, once the app is deployed we wouldn't really need to support the `localhost:3000` address, though it may be nice for debugging purposes to retain a possibility, hence the mentioning of plugging that through an env file.